plugins/lyrics-genius/menu.js

```diff
@@ -0,0 +1,17 @@
+const { setMenuOptions } = require("../../config/plugins");
```

Comment:
```suggestion
const { setOptions } = require("../../config/plugins");
```
Actually, the difference is that `setMenuoptions` will restart the app if the restart on config change setting is enabled

in this case, this is not needed since the option is synced with back.js (*and doesn't require a restart to apply*)

Reply:
oki